site/src/pages/components/switch.explainer.mdx

```diff
@@ -0,0 +1,438 @@
+- `drag&drop`: the switch can be toggled by dragging the thumb
+
+#### Form association
+
+<div class="question">
+  How should form validation work? Is there a way to describe that the valid state is "off" instead
+  of the more intuitive "on"?
+</div>
+
+A `<switch>` can be part of a form and has a mechanism (`type="submit"`) to immediately submit the form it's part of. The value of a switch is reported to the form target as either "on" or "off".
+
+#### Animation
+
+Two popular animation techniques are supported.
+
+##### Background fade
+
+Upon toggling the switch, it's background changes from grey (off) to green (on) and the toggle slides to the respective position. Content on the thumb can indicate the currently active state. Track content is also possible in this variant, but as the track does not move, the content is being hidden and revealed by the sliding thumb.
+
+##### Track slide
+
+Upon toggling the switch, the track and the thumb slide together to the respective position. The track reveals the active side of the track while the inactive track side is being hidden. This variant imitates physical switches and can also reveal track content on either side of the thumb, describing the currently active state.
+
+### Risks and Challenges
+
+{/* _Notable risks or challenges associated with the component. Link issues related to platform blockers._ */}
+
+Implementing the switch as a form element could lead developers to use it as a checkbox replacement.
+
+### Prior Art/Examples
+
+{/* _Link to the OpenUI research page for the component. If any existing, canonical, or exemplary implementations of the component are found beyond what is documented in the research page, make a separate PR to add them there._ */}
+Open UI research page: https://open-ui.org/components/switch/
+
+---
+
+## Design
+
+_Describe the design of the component, thinking through several perspectives:_
+
+- _A customer using the component on a web page._
+- _A developer building an app with the component and interacting through HTML/CSS/JavaScript._
+- _A designer customizing the component._
+
+### APIs
+
+_Outline the key elements of the component's public API surface, taking into consideration conformance guidelines. Make sure to discuss differences and rationalizations. Consider high and low-level APIs. Attempt to design a powerful and extensible low-level API with a high-level API for developer/designer ergonomics and simplicity._
+
+#### Properties and Attributes
+
+<div class="question">
+  If form associated, should default value of "on" be submitted if switch is on and the user did not
+  specify a value to align with the [checkbox default value
+  behavior](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#value:~:text=If%20the%20value%20attribute%20was%20omitted%2C%20the%20default%20value%20for%20the%20checkbox%20is%20on%2C%20so%20the%20submitted%20data%20in%20that%20case%20would%20be%20subscribe%3Don.)?
+</div>
+
+<div class="question">
```

Comment:
Can you please file issues for the open questions

Reply:
@gregwhitworth, I think I'll open issues sequentially, not all at the same time, because some questions are depending on decisions from other questions.